static/sass/_patterns_logo-links.scss

```diff
@@ -1,11 +1,14 @@
 @mixin p-logo-link {
-
   .p-logo-link {
     border: 1px solid transparent;
     border-radius: 2px;
     display: block;
     margin-top: 0;
 
+    img {
```

Comment:
You should target elements in patterns. It coupled the HTML to closely to the pattern.

Reply:
It bothers me how many times I need to add `display: block` to images in Vanilla. And adding a specific pattern element class name just to do that seemed overkill, but I agree that selectors targeting elements should be avoided, so I guess it will need to be `p-logo-link__image`.